Formula/libpeas.rb

```diff
@@ -10,18 +10,46 @@ class Libpeas < Formula
     sha256 "0f521913ca0eaf13b55aacb75e4b87a730be1f527215741ae2ba207caac523b2" => :el_capitan
   end
 
+  option "with-python", "install support for running Python plugins"
+
   depends_on "gettext" => :build
   depends_on "intltool" => :build
   depends_on "pkg-config" => :build
   depends_on "glib"
   depends_on "gobject-introspection"
   depends_on "gtk+3"
+  depends_on "python" => :optional
+  depends_on "python3" => :optional
+  depends_on "pygobject3" if build.with?("python")
+  depends_on "pygobject3" => ["--with-python3"] if build.with?("python3")
 
   def install
-    system "./configure", "--disable-dependency-tracking",
-                          "--disable-silent-rules",
-                          "--prefix=#{prefix}",
-                          "--enable-gtk"
+    args = %W[
+      --disable-dependency-tracking
+      --disable-silent-rules
+      --prefix=#{prefix}
+      --enable-gtk
+    ]
+
+    if build.with? "python"
+      # extra hoop to jump if both, Python 2 and 3 are installed in parallel
+      py2_prefix = Utils.popen_read("python2-config --prefix").chomp
+      py2_lib = "#{py2_prefix}/lib"
+      ENV.append "LDFLAGS", "-L#{py2_lib}"
+      # configure argument
+      args << "--enable-python2"
+    end
+
+    if build.with? "python3"
+      # extra hoop to jump if both, Python 2 and 3 are installed in parallel
+      py3_prefix = Utils.popen_read("python3-config --prefix").chomp
+      py3_lib = "#{py3_prefix}/lib"
```

Comment:
We know this variable since we're always using Homebrew's `python3` now. It's:
```ruby
Formula["python3"].opt_frameworks/"Python.framework/Versions/#{pyver}/lib
```

So let's just do:
```ruby
if build.with? "python3"
  py3_lib = Formula["python3"].opt_frameworks/"Python.framework/Versions/#{pyver}/lib
  ENV.append "LDFLAGS", "-L#{py3_lib}"
  args << "--enable-python3"
end
```

Reply:
Done with d7408de76d47c530ae70e4b51d299922607150a3. Some extra tweaking was necessary, however, to get the version part of the library path right.